Tests/Simulation/SimulatedLoadTests.cs

```diff
@@ -123,6 +127,99 @@ async Task ActAsync(SimulatedDevice device, TimeSpan startOffset)
             }
         }
 
+        /// <summary>
+        /// This test emulates a scenario where we have several plants with multiple concentrators and devices,
+        /// connected to several gateways (either on-premises or in the cloud).
+        /// </summary>
+        [Fact]
+        public async Task Connected_Factory_Load_Test_Scenario()
+        {
+            const int numberOfFactories = 1;
+            const double joinsPerSecond = 1.5;
+            var messagesPerSecond = MoreLinq.MoreEnumerable.Generate(1.5, old => old > 10 ? old : old + 2);
+            const int numberOfLoops = 5;
+            var stationsPerFactory = this.simulatedBasicsStations.Count / numberOfFactories;
+
+            // The total number of concentratos can be configured via the test configuration. It will de distributed evenly among factories;
```

Comment:
nit: typo:

```suggestion
            // The total number of concentrators can be configured via the test configuration. It will de distributed evenly among factories;
```